src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -273,6 +278,58 @@ EmberAfNetworkCommissioningError OnEnableNetworkCommandCallbackInternal(app::Com
     return err;
 }
 
+static void ThreadScanNetworksResultCallback(chip::DeviceLayer::ConnectivityManager::ThreadDiscoveryResult * result)
+{
+    if (result == nullptr)
+    {
+        return;
+    }
+
+    ChipLogProgress(DeviceLayer, "ThreadDiscoveryResult for scanned network index = %d:", sThreadNetworksScanned);
+    ChipLogProgress(DeviceLayer, "- discoveryResponseVersion: %d", result->discoveryResponseVersion);
+    ChipLogProgress(DeviceLayer, "- discoveryResponseNativeCommissionerFlag: %d", result->discoveryResponseNativeCommissionerFlag);
+    ChipLogProgress(DeviceLayer, "- extendedPanId: %02X%02X%02X%02X%02X%02X%02X%02X", result->extendedPanId[0],
+                    result->extendedPanId[1], result->extendedPanId[2], result->extendedPanId[3], result->extendedPanId[4],
+                    result->extendedPanId[5], result->extendedPanId[6], result->extendedPanId[7]);
+    ChipLogProgress(DeviceLayer, "- networkName: %s", result->networkName);
+    ChipLogProgress(DeviceLayer, "- steeringDataLength: %s", result->steeringDataLength);
```

Comment:
What is `steeringData`? Do we need it in the final response? If not I suggest to only extract fields which are defined in the Thread spec compliant Discovery Resepons

Reply:
Steering data describes joiners able to join the Thread network and according to Thread spec Steering Data TLV may be present in the Discovery Response.